rules/crawlers-user-agents.data

```diff
@@ -36,3 +36,532 @@ Owlin bot
+LeechGet
+LexiBot
+Lftp
+LibWeb
+Libwhisker
+Lightspeedsystems
+Likse
+Linguee
+Linkdexbot
+LinkextractorPro
+LinkpadBot
+LinkScan
+LinksManager
+LinkWalker
+LinqiaMetadataDownloaderBot
+LinqiaRSSBot
+LinqiaScrapeBot
+Lipperhey
+Litemage_walker
+Lmspider
+LNSpiderguy
+Ltx71
+Magnet
+Mag-Net
+magpie-crawler
+Mail.RU_Bot
+Majestic12
+MarkMonitor
+MarkWatch
+Masscan
+Mass Downloader
+Mata Hari
+MauiBot
+Meanpathbot
+Mediatoolkitbot
+mediawords
+MegaIndex.ru
+Metauri
+MFC_Tear_Sample
+Microsoft Data Access
+Microsoft URL Control
+MIDown tool
+MIIxpc
+Mister PiX
+MJ12bot
+Mojeek
+Morfeus Fucking Scanner
+Mr.4x3
+MSFrontPage
+MSIECrawler
+Msrabot
+MS Web Services Client Protocol
+muhstik-scan
+Musobot
+Name Intelligence
+Nameprotect
+Navroad
+NearSite
+Needle
+Nessus
+NetAnts
+Netcraft
+netEstate NE Crawler
+NetLyzer
+NetMechanic
+NetSpider
+Nettrack
+Net Vampire
+Netvibes
+NetZIP
+NextGenSearchBot
+Nibbler
+NICErsPRO
+Niki-bot
+Nikto
+NimbleCrawler
+Ninja
+Nmap
+NPbot
+Nutch
+oBot
+Octopus
+Offline Explorer
+Offline Navigator
+Openfind
+OpenLinkProfiler
+Openvas
+OpenVAS
+OrangeBot
+OrangeSpider
+OutclicksBot
+OutfoxBot
+PageAnalyzer
+Page Analyzer
+PageGrabber
+page scorer
+PageScorer
+Panscient
+Papa Foto
+Pavuk
+pcBrowser
+PeoplePal
+PHPCrawl
+Picscout
+Picsearch
+PictureFinder
+Pimonster
+Pi-Monster
+Pixray
+PleaseCrawl
+plumanalytics
+Pockey
+POE-Component-Client-HTTP
+Probethenet
+ProPowerBot
+ProWebWalker
+Psbot
+Pump
+PxBroker
+QueryN Metasearch
+Quick-Crawler
+RankActive
+RankActiveLinkBot
+RankFlex
+RankingBot
+RankingBot2
+Rankivabot
+RankurBot
+RealDownload
+Reaper
+RebelMouse
+Recorder
+RedesScrapy
+ReGet
+RepoMonkey
+Ripper
+RocketCrawler
+Rogerbot
+s1z.ru
+SalesIntelligent
+SBIder
+ScanAlert
+Scanbot
+scan.lol
+Scrapy
+Screaming
+ScreenerBot
+Searchestate
+SearchmetricsBot
+Semrush
+SemrushBot
+SEOkicks
+SEOlyticsCrawler
+Seomoz
+SEOprofiler
+seoscanners
+SeoSiteCheckup
+SEOstats
+sexsearcher
+Shodan
+Siphon
+SISTRIX
+Sitebeam
+SiteExplorer
+Siteimprove
+SiteLockSpider
+SiteSnagger
+SiteSucker
+Site Sucker
+Sitevigil
+Slackbot-LinkExpanding
+SlySearch
+SmartDownload
+SMTBot
+Snake
+Snapbot
+Snoopy
+SocialRankIOBot
+sogouspider
+Sogou web spider
+Sosospider
+Sottopop
+SpaceBison
+Spammen
+SpankBot
+Spanner
+Spbot
+Spinn3r
+SputnikBot
+Sqlmap
+Sqlworm
+Sqworm
+Steeler
+Stripper
+Sucker
+Sucuri
+SuperBot
+SuperHTTP
+Surfbot
+SurveyBot
+Suzuran
+Swiftbot
+Szukacz
+T0PHackTeam
+T8Abot
+tAkeOut
+Teleport
+TeleportPro
+Telesoft
+Telesphoreo
+Telesphorep
+The Intraformant
+TheNomad
+TightTwatBot
+Titan
+Toata
+Toweyabot
+Tracemyfile
+Trendiction
+Trendictionbot
+trendiction.com
+trendiction.de
+True_Robot
+Turingos
+Turnitin
+TurnitinBot
+TwengaBot
+Twice
+Typhoeus
+UnisterBot
+URLy.Warning
+URLy Warning
+Vacuum
+Vagabondo
+VB Project
+VCI
+VeriCiteCrawler
+VidibleScraper
+Virusdie
+VoidEYE
+Voil
+Voltron
+Wallpapers/3.0
+WallpapersHD
+WASALive-Bot
+WBSearchBot
+Webalta
+WebAuto
+Web Auto
+WebBandit
+WebCollage
+Web Collage
+WebCopier
+WEBDAV
```

Comment:
should we consider WebDav a "bad bot"? warning: `pmFromFile` performs a **case-insensitive** match. This will match `Microsoft-WebDAV-MiniRedir` for example.

Reply:
Removed